marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/saveSavedQuery.sjs

```diff
@@ -24,11 +24,9 @@ var saveQuery;
 var userCollections = ["http://marklogic.com/data-hub/saved-query"];
 var queryDocument = JSON.parse(saveQuery);
 
-let authorized = new Security().getRolesAndAuthorities().authorities.includes("canManageSavedQuery");
-
-if(!authorized) {
-    ds.throwForbidden(xdmp.getCurrentUser() + " user doesn't have authority to save or update query");
-}
+const sec = new Security();
+let authorized = new Security().getRolesAndAuthorities().authorities.includes("manageSavedQuery");
```

Comment:
Do you still need this line of code? I would think the sec.dataHubAuthorityAssert function would suffice.